godot_project/editor/input_handlers/molecular_structures/add_posing_atom_input_handler.gd

```diff
@@ -355,6 +358,8 @@ func _should_show() -> bool:
 		return false
 	if parameters.get_create_mode_type() != CreateObjectParameters.CreateModeType.CREATE_ATOMS_AND_BONDS:
 		return false
+	if _is_shortcut_pressed():
+		return true
```

Comment:
this is bypassing `_workspace_context.workspace.representation_settings.get_display_auto_posing()` i dont think that was  required or desired

Reply:
Currently, if you hold the alt key down, candidates will show up (if we're in create mode), regardless of was the display settings say. 
So, either the current behavior is wrong, either this line was missing to match the current behavior.

Reply:
Well, that's probably a sepparate bug that needs to be resolved then